index.html

```diff
@@ -33,6 +33,274 @@
+                  </p>
+                  <p>If a hardware error
+                    such as an OS/program/webpage lock prevents access to at least one device,
+                    <a>reject</a> <var>p</var> with a new
+                    {{DOMException}} object whose
+                    {{DOMException/name}} attribute has the value
+                    {{NotReadableError}} and abort these steps.
+                  </p>
+                  <p>If device access fails for
+                    any reason other than those listed above, <a>reject</a>
+                    <var>p</var> with a new {{DOMException}}
+                    object whose {{DOMException/name}} attribute has the
+                    value {{AbortError}} and abort these steps.
+                  </p>
+                </li>
+                <li>
+                  <p>Let <var>streams</var> be the
+                    list of {{MediaStream}} objects for which the permission was granted.</p>
+                </li>
+                <li>
+                  <p><a>Resolve</a> <var>p</var> with <var>streams</var> and
+                    abort these steps.</p>
+                </li>
+              </ol>
+            </li>
+            <li>
+              <p>Return <var>p</var>.</p>
+            </li>
+          </ol>
+          <p>
+            The <a href="#dfn-user-agent">user agent</a> MUST return a list of {{MediaStream}}
+            objects, where each stream belongs to exactly one of the
+            <a data-cite="screen-capture#dfn-monitor">monitors</a> attached to the device.
+          </p>
+          <p>
+            Information that is not currently rendered to the screen SHOULD be obscured in captures
+            unless the application has been specifically authorized to access that content (e.g.
+            through means such as
+            <a data-cite="screen-capture#elevated-permissions">elevated permissions</a>).
+          </p>
+        </dd>
+      </dl>
+    </section>
+
+    <section>
+      <h2>ScreenCaptureMediaStreamTrack</h2>
+      {{MediaDevices/ScreenCaptureMediaStreamTrack}} extends {{MediaStreamTrack}} and
+      corresponds to a <a data-cite="screen-capture#dfn-monitor">monitor</a>.
+      It additionally provides a connection to the window placement API [[!window-placement]]
+      with the {{MediaDevices/ScreenCaptureMediaStreamTrack/screenDetailed}} function with which metadata about the
+      captured <a data-cite="screen-capture#dfn-monitor">monitor</a> can be retrieved.
+      <pre class="idl">
+        [Exposed=Window]
+        interface ScreenCaptureMediaStreamTrack : MediaStreamTrack {
+          ScreenDetailed screenDetailed();
+        };
+      </pre>
+
+      <dl data-link-for="ScreenCaptureMediaStreamTrack" data-dfn-for="ScreenCaptureMediaStreamTrack" class="methods">
+        <dt>
+          <dfn>screenDetailed</dfn>
+        </dt>
+        The {{screenDetailed()}} method returns a {{ScreenDetailed}} object [[!window-placement]]
```

Comment:
Rephrase it as specification rather than documentation.

Reply:
"returns" --> "MUST return"